ui-components/src/locales/general.json

```diff
@@ -1106,5 +1106,37 @@
     "applicantsWillBeContacted": "Applicants will be contacted by the property agent in rank order until vacancies are filled.",
     "allInfoWillBeVerified": "All of the information that you have provided will be verified and your eligibility confirmed. Your application will be removed from the waitlist if you have made any fraudulent statements. If we cannot verify a housing preference that you have claimed, you will not receive the preference but will not be otherwise penalized.",
     "bePreparedIfChosen": "Should your application be chosen, be prepared to fill out a more detailed application and provide required supporting documents."
+  },
+  "listingFilters":{
+    "modelTitle":"Filter Results",
+    "modelHeader":"Use these options to refine your list of properties.",
+    "unitOptions":{
+      "label": "Unit Size:",
+      "description":"",
```

Comment:
Sorry I forgot to leave this comment before, but we should remove these empty "description" strings.